<?xml version="1.0"?>
<launch>
  <arg name="cam_name" default="csi_cam_0" />
  <arg name="topic_name" default="cv_video" />

  <node pkg="jetracer" type="cv_video.py" name="cv_video" output="screen">
        <param name="camera_name"  value="$(arg cam_name)" />
	<param name="topic_name"  value="$(arg topic_name)" />
   </node>

</launch>

